### PR TITLE
Support command line parameters for azure gradle plugins

### PR DESCRIPTION
**Pull request by Flanker32**
What does this implement/fix? Explain your changes.
---------------------------------------------------
Support command line parameters for azure gradle plugins
- Support read parameters from command line in webapp/function gradle plugin
- Add init script for webapp/function gradle plugin

Now we could deploy gradle projects to azure without modify origin build file in project
``` bash
curl https://raw.githubusercontent.com/microsoft/azure-gradle-plugins/hanli/parameters/azure-functions-gradle-plugin/init.gradle -o init.gradle
gradle --init-script init.gradle azureFunctionsDeploy -DappName=<testAppName> -DresourceGroup=<testResourceGroup>
```

todo: only apply plugin if user did not apply plugin before

Does this close any currently open issues?
------------------------------------------

Has this been tested?
---------------------------
- [ ] Tested

### CODE REVIEW

File: azure-gradle-plugins-common/src/main/resources/bundles/com/microsoft/azure/toolkit/operation.properties [hunk 1]
```diff
@@ -1,68 +1,8 @@
...
-webapp.update_slot.slot|app=update deployment slot ({0}) of Azure Web App ({1}) from config
-webapp.delete_app.app=delete Azure Web App ({0})
-webapp.start_app.app=start Azure Web App ({0})
-webapp.stop_app.app=stop Azure Web App ({0})
-webapp.restart_app.app=restart Azure Web App ({0})
-webapp.delete_slot.slot|app=delete deployment slot ({0}) of Azure Web App ({1})
-webapp.start_slot.slot|app=start deployment slot ({0}) of Azure Web App ({1})
-webapp.stop_slot.slot|app=stop deployment slot ({0}) of Azure Web App ({1})
-webapp.restart_slot.slot|app=restart deployment slot ({0}) of Azure Web App ({1})
+user/functionapp.package=generate configuration files and prepare staging directory
```
Comment (Flanker32) [inline azure-gradle-plugins-common/src/main/resources/bundles/com/microsoft/azure/toolkit/operation.properties:1]: add up properties from azure-toolkit-lib

**Review by wangmingliang-ms — APPROVED** [review]
LGTM

File: azure-functions-gradle-plugin/src/main/java/com/microsoft/azure/plugin/functions/gradle/util/GradleProjectUtils.java [hunk 2]
```diff
@@ -14,15 +17,17 @@
...
 import java.util.List;
 import java.util.Objects;
+import java.util.stream.Collectors;
 
 /**
  * Convert a Gradle project to a common project
  */
 public class GradleProjectUtils {
     private static final String MAIN_SOURCE_SET_NAME = "main";
+    public static final String DEPENDENCY_WARNING = "Following dependencies could not be founded, " +
```
Comment (wangmingliang-ms) [inline azure-functions-gradle-plugin/src/main/java/com/microsoft/azure/plugin/functions/gradle/util/GradleProjectUtils.java:29]: found

File: azure-functions-gradle-plugin/src/main/java/com/microsoft/azure/plugin/functions/gradle/util/GradleProjectUtils.java [hunk 3]
```diff
@@ -14,15 +17,17 @@
...
+            "please check whether related modules has been packaged \n %s";
```
Comment (wangmingliang-ms) [inline azure-functions-gradle-plugin/src/main/java/com/microsoft/azure/plugin/functions/gradle/util/GradleProjectUtils.java:30]: have

**Review by wangmingliang-ms — APPROVED** [review]
LGTM

**Pull request merged**